go.mod

```diff
@@ -25,36 +25,36 @@ require (
 	github.com/headzoo/ut v0.0.0-20181013193318-a13b5a7a02ca // indirect
 	github.com/integr8ly/cloud-resource-operator v0.42.0
 	github.com/integr8ly/keycloak-client v0.1.11
-	github.com/onsi/gomega v1.20.1
+	github.com/laurafitzgerald/observability-operator/api/v4 v4.0.0-20230104125505-f1e38a9832b4
+	github.com/onsi/gomega v1.24.2
 	github.com/openshift/api v3.9.1-0.20191031084152-11eee842dafd+incompatible
 	github.com/openshift/cluster-samples-operator v0.0.0-20191113195805-9e879e661d71
 	github.com/operator-framework/api v0.17.1
 	github.com/operator-framework/operator-lifecycle-manager v0.21.1
 	github.com/operator-framework/operator-registry v1.26.2
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.58.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.61.1
 	github.com/prometheus/alertmanager v0.22.0
-	github.com/prometheus/client_golang v1.13.0
-	github.com/redhat-developer/observability-operator/v3 v3.0.16
+	github.com/prometheus/client_golang v1.14.0
 	github.com/sirupsen/logrus v1.8.1
-	golang.org/x/net v0.0.0-20221004154528-8021a29435af
+	golang.org/x/net v0.4.0
 	golang.org/x/sync v0.0.0-20220929204114-8fcdb60fdcc0
-	golang.org/x/term v0.0.0-20220722155259-a9ba230a4035
+	golang.org/x/term v0.3.0
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.3
-	k8s.io/apiextensions-apiserver v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20220803164354-a70c9af30aea // indirect
-	k8s.io/utils v0.0.0-20220922133306-665eaaec4324
-	sigs.k8s.io/controller-runtime v0.12.3
+	k8s.io/api v0.26.0
```

Comment:
I think I'm more in favor of either removing deps from other operators in one PR and bumping RHOAMs k8s.io to latest, or bump in other operators first and then in RHOAM to avoid introducing mass replaces.
WDYT?